Microsoft.Toolkit.Uwp.SampleApp/project.json

```diff
@@ -1,5 +1,6 @@
 ﻿{
   "dependencies": {
+    "Microsoft.Graph": "1.1.1",
```

Comment:
@deltakosh I needed to add Microsoft.Graph reference in the SamplesApp either way I could not be able to compile the app.. (using Visual Studio 2017 Community RC 15.0.26020.0).